engine-tests/src/tests/uniswap.rs

```diff
@@ -54,7 +54,7 @@ fn test_uniswap_exact_output() {
 
     let (_result, profile) =
         context.add_equal_liquidity(LIQUIDITY_AMOUNT.into(), &token_a, &token_b);
-    test_utils::assert_gas_bound(profile.all_gas(), 58);
+    test_utils::assert_gas_bound(profile.all_gas(), 59);
```

Comment:
Why does this change?

Reply:
Wasn't needed. The test was failing for me locally, but it was just a difference between my local solidity artifacts and the one in CI. Recompiling them locally fix it. Change removed from diff.